### PR TITLE
Option to strictly/leniently handle SMDB errors 

### PR DESCRIPTION
**Pull request by vladsaveliev**
Just a handy option that should never be used, but sometimes useful for debug purposes.

### CODE REVIEW

File: cpg_pipes/pipeline/cli_opts.py [hunk 1]
```diff
@@ -224,6 +224,13 @@ def main(custom_argument: str):
             'slack_channel',
             help='Slack channel to send status reports with CPG status reporter.',
         ),
+        click.option(
+            '--crash-on-db-erorrs/--lenient-on-db-erorrs',
+            'crash_on_db_errors',
```
Comment (lgruen) [inline cpg_pipes/pipeline/cli_opts.py:229]:
```suggestion
            '--smdb-errors-are-fatal/--no-smdb-errors-are-fatal',
            'smdb_errors_are_fatal',
```

File: cpg_pipes/providers/cpg/inputs.py [hunk 2]
```diff
@@ -111,15 +120,40 @@ def populate_alignment_inputs(self, cohort: Cohort) -> None:
...
+            seq_infos: list[dict] = self.db.seqapi.get_sequences_by_sample_ids(
+                cohort.get_sample_ids()
+            )
+        except ApiException:
+            if self.crash_on_erorrs:
+                raise
+            else:
+                logger.error(
+                    'Getting sequencing data from SMDB resulted in an error. '
+                    'Continueing without sequencing data because lenient=true. '
```
Comment (lgruen) [inline cpg_pipes/providers/cpg/inputs.py:134]:
```suggestion
                    'Continuing without sequencing data because of flag override. '
```

**Review by lgruen — APPROVED** [review]
Suggested renaming the flag slightly, otherwise looks good!

**Comment by vladsaveliev** [comment]
Argh, merged into wrong branch :(

**Pull request merged**